test_regress/t/t_mod_topmodule.pl

```diff
@@ -2,18 +2,16 @@
 if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
 # DESCRIPTION: Verilator: Verilog Test driver/expect definition
 #
-# Copyright 2008 by Wilson Snyder. This program is free software; you
+# Copyright 2021 by Wilson Snyder. This program is free software; you
 # can redistribute it and/or modify it under the terms of either the GNU
 # Lesser General Public License Version 3 or the Perl Artistic License
 # Version 2.0.
 # SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
 
-scenarios(vlt => 1);
+scenarios(linter => 1);
```

Comment:
Could you please still use vlt =>, then also call execute() below.

While unlikely that there's a bug, I think we should test that this new use of top_module works at all stages, not just lint.